Zotero/Controllers/Database/Requests/MoveItemsToParentDbRequest.swift

```diff
@@ -41,6 +41,7 @@ struct MoveItemsToParentDbRequest: DbRequest {
         }
 
         // Update the parent item, so that it's updated in the item list to show attachment/note marker
+        parent.baseTitle = parent.baseTitle
```

Comment:
There was a bug in Realm some long time ago which caused observer not being notified when a child item was added/deleted. So we had this workaround that we would edit the parent item so that it gets the notification of a change. It was fixed some time ago (even though I forgot to remove those comments apparently) but it now seems to have resurfaced. 

So pretty much when you add/delete a child item, the parent doesn't get a "modified" notification so the item list was not updated properly.